walkthroughs/howto-cross-ekscluster-appmesh-with-cloudmap/README.md

```diff
@@ -0,0 +1,232 @@
+
+So, Lets get started..
+
+## Prerequisites
+
+In order to successfully carry out the base deployment:
+
+* Make sure to have newest AWS CLI (https://aws.amazon.com/cli/) installed, that is, version 1.16.268 or above.
+* Make sure to have kubectl installed (https://kubernetes.io/docs/tasks/tools/install-kubectl/), at least version 1.11 or above.
+* Make sure to have jq installed (https://stedolan.github.io/jq/download/).
+* Make sure to have aws-iam-authenticator installed (https://github.com/kubernetes-sigs/aws-iam-authenticator), required for eksctl
+* Install eksctl (https://eksctl.io/), for example, on macOS with brew tap weaveworks/tap and brew install weaveworks/tap/eksctl, and make sure it's on at least on version 0.1.26.
+
+Note that this walkthrough assumes throughout to operate in the us-east-1 region.
+
+### Cluster provisioning
+
+Create an EKS cluster with eksctl using the following command:
+```
+eksctl create cluster --name=eksc2 --nodes=3 --alb-ingress-access 
+--region=us-east-1 --ssh-access --asg-access  --full-ecr-access  
+--external-dns-access --appmesh-access --vpc-cidr 172.16.0.0/16
+--auto-kubeconfig
+#[✔]  EKS cluster "eksc2-useast1" in "us-east-1" region is ready
+```
+
+Once cluster creation is complete open an other tab and create an other EKS cluster with eksctl using the following command:
+Note: Use the public and private subnets created as part of cluster1 in this command. See this (https://eksctl.io/usage/vpc-networking/) for more details.
+```
+eksctl create cluster --name=eksc1 --nodes=2 --alb-ingress-access 
+--region=us-east-1 --ssh-access --asg-access  --full-ecr-access  
+--external-dns-access --appmesh-access  --auto-kubeconfig 
+--vpc-private-subnets=<comma seperated private subnets>
+--vpc-public-subnets=<comma seperated public subnets>
+#[✔]  EKS cluster "eksc1" in "us-east-1" region is ready
+```
+
+When completed, update the KUBECONFIG environment variable in each tab according to the eksctl output, repectively:
+```
+export KUBECONFIG=~/.kube/eksctl/clusters/eksc2
+export KUBECONFIG=~/.kube/eksctl/clusters/eksc1 
+Note: Do this respective tabs
+```
+
+You have now setup the two clusters and pointing kubectl to respective clusters. Congratulations.
+
+### Deploy AppMesh Custom Components
+
+In order to automatically inject AppMesh components and proxies on pod creation we need to create some custom resources on the clusters. We will use *helm* for that. We need install tiller on both the clusters and run the following commands on both clusters for that.
+
+*Code base*
+
+Clone the repo and cd into the appropriate directory. We will be running all commands from this path.
+```
+>> git clone https://github.com/aws/aws-app-mesh-examples (https://github.com/aws/aws-app-mesh-examples).git
+>> cd aws-app-mesh-examples/walkthroughs/howto-cross-ekscluster-appmesh-with-cloudmap/
+```
+
+*Install Helm*
```

Comment:
Instead of here, can you please update [EKS walkthrough](https://github.com/aws/aws-app-mesh-examples/tree/master/walkthroughs/eks) and refer to it from here. That way other guides can also benefit from this. And will also partially address https://github.com/aws/aws-app-mesh-roadmap/issues/128.

Reply:
Done, made changes in base.md and referencing from here.